app/services/gpturk_service.rb

```diff
@@ -2,11 +2,27 @@ module GpturkService
   require 'rest-client'
   require 'json'
 
+  def self.is_this_spam?(text)
```

Comment:
Sorry, but I should have suggested a more conventional `is_spam?` name.

Reply:
of course, lol!